third_party/move/move-compiler-v2/tests/checking/typing/native_structs_pack_unpack.exp

```diff
@@ -1,23 +1,17 @@
 
 Diagnostics:
-bug: inconsistent struct definition
+error: native struct `C::T` has no fields
```

Comment:
I'm just turning an existing 'bug' error message into a more reasonable ons. Native structs are not supported since a long time, I was surprised to find a test for them. We should probably add an error check elsewhere but it is unrelated to this PR. There are no docs because this feature isn't really present. (At the latest the VM will bail on this.)